pkgs/development/python-modules/selectolax/default.nix

```diff
@@ -0,0 +1,48 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchFromGitHub,
+  cython,
+  setuptools,
+  pytestCheckHook,
+}:
+
+buildPythonPackage rec {
+  pname = "selectolax";
+  version = "0.4.0";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "rushter";
+    repo = "selectolax";
+    rev = "v${version}";
+    fetchSubmodules = true;
```

Comment:
Can we unvendor lexbor and maybe modest?

Reply:
It uses an unreleased version of lexbor.

Reply:
I will take a look tomorrow.